hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/service/OpenKeyCleanupService.java

```diff
@@ -82,9 +82,11 @@ public class OpenKeyCleanupService extends BackgroundService {
   private final AtomicBoolean suspended;
 
   public OpenKeyCleanupService(long interval, TimeUnit unit, long timeout,
-      OzoneManager ozoneManager, ConfigurationSource conf) {
+                               OzoneManager ozoneManager,
+                               ConfigurationSource conf) {
     super("OpenKeyCleanupService", interval, unit,
-        OPEN_KEY_DELETING_CORE_POOL_SIZE, timeout);
+        OPEN_KEY_DELETING_CORE_POOL_SIZE, timeout,
+        ozoneManager.getNodeDetails().threadNamePrefix());
```

Comment:
similar changes is missing for org.apache.hadoop.ozone.om.service.SnapshotDiffCleanupService#SnapshotDiffCleanupService

Reply:
Done!